### PR TITLE
added discussed changes of the last sequencing round to the protokoll

### PR DESCRIPTION
**Pull request by teresa-m**

### CODE REVIEW

File: _protocols/beer-dna-sequencing.md [hunk 1]
```diff
@@ -28,18 +55,19 @@ The most important: **7.5µl DNA** extracted as described in the [DNA extraction
 - Fragmentation Mix (FRA)
```
Comment (erxleben) [inline _protocols/beer-dna-sequencing.md]:
```suggestion
- Fragmentation Mix (FRA) from sequencing kit
  Random fragmentation of genomic DNAs is usually done by enzyme treatments
```

File: _protocols/beer-dna-sequencing.md [hunk 2]
```diff
@@ -28,18 +55,19 @@ The most important: **7.5µl DNA** extracted as described in the [DNA extraction
...
 - Rapid Adapter (RAP)
```
Comment (erxleben) [inline _protocols/beer-dna-sequencing.md]:
```suggestion
- Rapid Adapter (RAP)
 Adapters are ligated to the DNA. They are needed for sequencing.
```

File: _protocols/beer-dna-sequencing.md [hunk 3]
```diff
@@ -28,18 +55,19 @@ The most important: **7.5µl DNA** extracted as described in the [DNA extraction
...
 - Nuclease-free water (e.g. ThermoFisher, cat #AM9937)
```
Comment (erxleben) [inline _protocols/beer-dna-sequencing.md]:
```suggestion
- Nuclease-free water (e.g. ThermoFisher, cat #AM9937)
 "Normal" water contains nucleases which may degrade DNA. 
```

File: _protocols/beer-dna-sequencing.md [hunk 4]
```diff
@@ -28,18 +55,19 @@ The most important: **7.5µl DNA** extracted as described in the [DNA extraction
...
-- 0.2 ml thin-walled PCR tubes
+- 0.2 ml PCR tubes
 - Ice
 
 ##### Needed material
 - Thermal cycler at 30° C and 80° C
```
Comment (erxleben) [inline _protocols/beer-dna-sequencing.md]:
```suggestion
- Thermocycler at 30° C and 80° C
 A Thermocycler is a PCR machine to amplify DNA. Denaturation of DNA takes place at 80°C, annealing primers to target sequence at 30°C. 
```

Comment (erxleben) [inline _protocols/beer-dna-sequencing.md]:
```suggestion
- Thermocycler at 30° C and 80° C
```

File: _protocols/beer-dna-sequencing.md [hunk 5]
```diff
@@ -57,8 +85,8 @@ The most important: **7.5µl DNA** extracted as described in the [DNA extraction
 - Flush Buffer (FB)
 - Loading Beads (LB)
 - Nuclease-free water (e.g. ThermoFisher, cat #AM9937)
-- 1.5 ml Eppendorf DNA LoBind tubes
-- Flongle device - flow cell and adapter
+- 1.5 ml Eppendorf DNA 
+- MinIon diveice including Flow cell
```
Comment (erxleben) [inline _protocols/beer-dna-sequencing.md]:
```suggestion
- MinIon device including Flow cell
```

File: _protocols/beer-dna-sequencing.md [hunk 6]
```diff
@@ -67,21 +95,24 @@ The most important: **7.5µl DNA** extracted as described in the [DNA extraction
 - P1000 pipette and tips
 - P20 pipette and tips
 - P10 pipette and tips
+- Vortex
+- centrifuge
 
 
-#### Prepare loading port
 
+#### Prepare substance of the priming mix
```
Comment (erxleben) [inline _protocols/beer-dna-sequencing.md]:
```suggestion
#### Prepare priming mix
```

File: _protocols/beer-dna-sequencing.md [hunk 7]
```diff
@@ -67,21 +95,24 @@ The most important: **7.5µl DNA** extracted as described in the [DNA extraction
...
-3. Open priming port to check for small bubbles
-4. If there are bubbles remove them by taking some liquid from the port
-5. To remove use the P1000 pipette set to 200 µl
-6. Remove the liquid by turning the weel but only until 220-230 µl
-7. Removing more than 30 µl will damage the pores in the array because they need to be covered by the buffer at all times
+
+#### Prepare loading port
+1. Open priming port to check for small bubbles
+2. To remove bubbles take some liquid from the port by use the P1000 pipette set to 200 µl
+3. Remove the liquid by turning the weel but only until 220-230 µl
```
Comment (erxleben) [inline _protocols/beer-dna-sequencing.md]: dont understand this. The wheel of the pipet?

**Comment by erxleben** [comment]
very first round done. But needs more improvement. Will work on it again

**Comment by bebatut** [comment]
I did some rearrangement of the content and also some formatting to support the PDFs export in different version. Let me know if the changes are ok for you, otherwise I can revert them

File: _protocols/beer-dna-sequencing.md [hunk 8]
```diff
@@ -6,126 +6,161 @@ image: /images/protocols/beer-dna-sequencing.jpg
...
-### Library preparation
-##### Needed consumables
-- ~400 ng high molecular weight genomic DNA
-- Fragmentation Mix (FRA)
-- Rapid Adapter (RAP)
+- Flow cell or Flongle
+- [Rapid Sequencing Kit](https://store.nanoporetech.com/sample-prep/rapid-sequencing-kit.html) including
+    - 30 µl Flush Tether (FLT)
+    - 34 μl Sequencing Buffer (SQB)
+    -  µl Flush Buffer (FB)
```
Comment (bebatut) [inline _protocols/beer-dna-sequencing.md]: How much quantity is needed there?

File: _protocols/beer-dna-sequencing.md [hunk 9]
```diff
@@ -6,126 +6,161 @@ image: /images/protocols/beer-dna-sequencing.jpg
...
+    3. Remove the liquid by turning the wheel of the pipet but only until 220-230 µl
 
-Requirements:
-- You have to have a account at nanopor MinION to start the software
-- The divice need to be attached to the computer to start the software
+    Note: removing more than 30 µl will damage the pores in the array because they need to be covered by the buffer at all times
 
-Either you find the "MinION Software" botton directly or you open it via the terminal using the path "/opt/ui/MinKNOW" 
+3. Prepare the **priming mix**
+    1. Add 30 µl of Flush Tether (FLT) directly to the Flush Buffer (FB) Eppi tube
```
Comment (bebatut) [inline _protocols/beer-dna-sequencing.md]: Which quantity of FB is needed?

File: _protocols/beer-dna-sequencing.md [hunk 10]
```diff
@@ -6,126 +6,161 @@ image: /images/protocols/beer-dna-sequencing.jpg
...
+    3. Spin down the FLT tube, mix by pipetting, and return to ice
+2. Prepare loading port
+    1. Open priming port on the flowcell to check for small bubbles
+    2. Remove possible bubbles by taking 200 µl liquid from the port using the P1000 pipette
+    3. Remove the liquid by turning the wheel of the pipet but only until 220-230 µl
 
-Requirements:
-- You have to have a account at nanopor MinION to start the software
-- The divice need to be attached to the computer to start the software
+    Note: removing more than 30 µl will damage the pores in the array because they need to be covered by the buffer at all times
```
Comment (bebatut) [inline _protocols/beer-dna-sequencing.md]: Given before it is written 200, I am not sure to understand how we can not remove more than 30 here

File: _protocols/beer-dna-sequencing.md [hunk 11]
```diff
@@ -6,126 +6,161 @@ image: /images/protocols/beer-dna-sequencing.jpg
...
+    1. Gently lift the SpotON sample port cover to make the SpotON sample port accessible
+    2. Load 200 μl of the priming mix into the flow cell via the **priming port (not the SpotON sample port)**, avoiding introduction of air bubbles
+    3. Mix the prepared library gently by pipetting up and down prior to loading
+    4. Add 75 μl of the library preparation to the flow cell via the SpotON sample port in a dropwise fashion
+    
+        Ensure each drop flows into the port before adding the next
 
+    5. Gently replace the SpotON sample port cover
+    6. Making sure the bung enters the SpotON port
+    7. Close the priming port and the MinION lid
```
Comment (bebatut) [inline _protocols/beer-dna-sequencing.md]: Will the sequencing start automatically after? What happen in the software?
We should probably add also a screenshot of the software but also way to check that everything is going well

File: _protocols/beer-dna-sequencing.md [hunk 12]
```diff
@@ -6,126 +6,161 @@ image: /images/protocols/beer-dna-sequencing.jpg
...
-3. Open priming port to check for small bubbles
-4. If there are bubbles remove them by taking some liquid from the port
-5. To remove use the P1000 pipette set to 200 µl
-6. Remove the liquid by turning the weel but only until 220-230 µl
-7. Removing more than 30 µl will damage the pores in the array because they need to be covered by the buffer at all times
+3. Select the flow cell type as written on the flow cell package
+4. Click on **"Check flow cell"** and then **"Start test"**
 
-#### Prepare flowcell priming mix
+<!--Add a screenshot-->
```
Comment (bebatut) [inline _protocols/beer-dna-sequencing.md]: Do you have a screenshot of the software here?

**Comment by bebatut** [comment]
I did quite some changes given the last discussions:

- Duplicate requirements to have them at the beginning of the protocol but also when needed
- Customize content for minion or flongle flowcell
- Expand the steps with details (given the content on the Nanopore website and external resources) and schemas

I think this protocol is now ready for review.

The different PDF can be downloaded there for review: https://send.firefox.com/download/2ca94dfb1552e9f5/#sZoUQCifeM4wGFhjscSWYw

**Pull request merged**